apps/teams-test-app/src/App.tsx

```diff
@@ -182,6 +183,7 @@ const App = (): ReactElement => {
         <VideoAPIs />
         <VideoExAPIs />
         <VisualMediaAPIs />
+        <CookieAccessComponent />
```

Comment:
It would be great to follow the alphabetical order.

Reply:
Updated.